.idea/vcs.xml

```diff
@@ -2,6 +2,5 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="" vcs="Git" />
-    <mapping directory="$PROJECT_DIR$/src/test/resources/network/brightspots/rcv/test_data" vcs="Git" />
```

Comment:
Do you know why this change is necessary? Not a blocker, but I'm curious.

Reply:
This tells intelliJ that the submodule is a submodule which enables Intellij vcs integration.

Reply:
it's pretty solid btw - worth checking out even if you have a git client you like. @